ouroboros-network/src/Ouroboros/Network/BlockFetch.hs

```diff
@@ -97,11 +97,13 @@ import           Control.Tracer (Tracer)
 import           Ouroboros.Network.AnchoredFragment (AnchoredFragment (..))
 import           Ouroboros.Network.Block
 import           Ouroboros.Network.DeltaQ
-                   ( PeerGSV(..), ballisticGSV, degenerateDistribution )
+                   ( PeerGSV(..), ballisticGSV, degenerateDistribution
+                   , SizeInBytes )
 
-import           Ouroboros.Network.BlockFetch.Client
 import           Ouroboros.Network.BlockFetch.State
-import           Ouroboros.Network.BlockFetch.Types
+import           Ouroboros.Network.BlockFetch.ClientRegistry
+                   ( FetchClientRegistry, newFetchClientRegistry
+                   , readFetchClientsStatus, readFetchClientsStateVars )
 
```

Comment:
It would nice if you re-export `bracketFetchClient`, `FetchMode (..)`, and `SizeInBytes` in `Ouroboros.Network.BlockFetch`, so users of Block Fetch (like `Ouroboros.Consensus.Node`) can just import `Ouroboros.Network.BlockFetch` and have everything they need, instead of requiring 3 imports.

Reply:
:+1:

Reply:
I have pushed a commit doing this to #474.